include/linker/linker-defs.h

```diff
@@ -51,11 +51,22 @@
  * ensure the objects aren't discarded if there is no direct reference to them
  */
 
+#if defined(__MWDT_LINKER_CMD__)
+
+#define MWDT_STRINGIFY(x) #x
+
+#define DEVICE_INIT_LEVEL(level)				\
+		__device_##level##_start = .;			\
+		KEEP(*(MWDT_STRINGIFY(.init_##level[0-9].*)));		\
+		KEEP(*(MWDT_STRINGIFY(.init_##level[1-9][0-9].*)));
+#else
```

Comment:
Does MWDT not support SORT?

What is the problem here?

Reply:
MWDT does not support SORT like ld.

Reply:
Does it happen to SORT by default? Because if not, then surely the devices will
be initialized in the wrong order and break?